pkg/apis/kubermatic/v1/external_cluster.go

```diff
@@ -96,7 +97,11 @@ type ExternalClusterKubeOneCloudSpec struct {
 	// ProviderName is the name of the cloud provider used, one of
 	// "aws", "azure", "digitalocean", "gcp",
 	// "hetzner", "nutanix", "openstack", "packet", "vsphere" KubeOne natively-supported providers
-	ProviderName         string                                  `json:"providerName"`
+	ProviderName string `json:"providerName"`
+
+	// Region is the kubernetes control plane region.
+	Region string `json:"region,omitempty"`
```

Comment:
What is this field supposed to say? Is this a cloud provider region? Is this merely information for the purpose of displaying it? I think the description on this needs to explain it better, because as an enduser I would have no idea what I need to put here.

Reply:
In addition, there doesn't seem to be information in the PR description on how those fields will be used. Can you please update the PR description to explain why this PR is happening?